jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/udp/controller/UdpServicesController.java

```diff
@@ -53,6 +54,9 @@ public UdpServicesController(OdeProperties odeProps) {
       // PSM internal port
       rm.submit(new PsmReceiver(odeProps));
```

Comment:
If we use this GenericReceiver, do we need to submit the above specialized receivers?

Reply:
Yes, the generic receiver is in addition to the existing receivers. At this point, I would recommend that we use the specific receivers whenever possible. The Generic Receiver should only be used in deployments where we are unable to separate which messages get sent where. For example, Cohda MK5 RSU units are only able to port forward all messages to a single port right now. Using the specific receivers will be slightly more performant, and will help prevent possible errors caused by message misidentification. Also removing the existing receivers would be a breaking change for nearly all of our deployments.